src/pointer.go

```diff
@@ -1,22 +1,27 @@
 package main
 
-import (  
-    "fmt"
+import (
+	"fmt"
 )
 
-func main() {  
-    b := 255
-    var a *int = &b                       //Saving b's address into variable a
```

Comment:
Thank you @mmichaelb  for the valuable contribution.
I think its better so start with basic things first like how to  declare a pointer, what is the significance of '*' and '&' sign. So please add some code comments related to that.

Reply:
I definitely agree with you on this one. However, my pull request/referred issue was really only about the implementation of the pointer function to show another Pointer related feature. Anyway, I am happy to deal with this matter as part of another issue.